.github/scripts/common.sh

```diff
@@ -17,15 +17,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 function tuttest_exec() {
```

Comment:
Why is this removed?

Reply:
Because it makes the logs completely unreadable

Reply:
Adding WIP tag btw, we are still experimenting here

Reply:
This is back in a more concise form.